kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActionsIterator.java

```diff
@@ -218,4 +225,31 @@ public void close() throws IOException {
             }
         };
     }
+
+    /**
+     * Given a checkpoint file, retrieve all the files that are part of the same checkpoint.
+     * <p>
+     * This is done by looking at the file name and finding all the files that have the same
+     * version number.
+     */
+    private CloseableIterator<FileStatus> retrieveRemainingCheckpointFiles(
+            FileStatus checkpointFile,
+            long version) {
+
+        // Filter out all the files that are not part of the same checkpoint
```

Comment:
What does this mean?

Reply:
updated the comment. It should have been: `// Find the contiguous parquet files that are part of the same checkpoint`